bench-vortex/src/bin/tpch_benchmark.rs

```diff
@@ -209,8 +209,20 @@ async fn bench_main(
             }
             let fastest = measures.iter().cloned().min().unwrap();
 
+            let storage = match url.scheme() {
+                "s3" => "s3",
+                "gcs" => "gcs",
+                "file" => "nvme",
+                otherwise => {
+                    println!("unknown URL scheme: {}", otherwise);
+                    return ExitCode::FAILURE;
```

Comment:
nit - there must be a nicer way, maybe `supported @ "s3" | "gcs" | "file" => support.to_owned() ...`

Reply:
I want `file` to become `nvme` though :/